src/FondOfSpryker/Shared/ThirtyFiveUp/Transfer/thirty_five_up.transfer.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<transfers xmlns="spryker:transfer-01"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="spryker:transfer-01 http://static.spryker.com/transfer-01.xsd">
+
+    <transfer name="ThirtyFiveUpOrder">
+        <property name="id" type="int"/>
+        <property name="items" singular="item" type="ThirtyFiveUpOrderItem[]"/>
+        <property name="salesOrderId" type="int"/>
```

Comment:
Is that the usual way of writing ? Or is it idSalesOrder..

Reply:
Ist egal, kann ich aber ändern